pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -651,6 +651,8 @@ func parseAppInstanceConfig(getconfigCtx *getconfigContext,
 		appinstancePrevConfigHash, configHash, Apps)
 	appinstancePrevConfigHash = configHash
 
+	devUUIDStr, _ := os.Hostname()
```

Comment:
please, add an error handler here....

Reply:
actually, AFAIU we get the UUID from the cloud and then set the hostname with it: https://github.com/lf-edge/eve/blob/master/pkg/pillar/cmd/client/client.go#L421 so is this assignment really needed here?

Reply:
Yes that is needed to see if the designated node id in the config is same as this current node. This same config goes to all the nodes in the cluster. Using already set UUID and comparing it with the one in the config, a node will find if it is designated or not.